src/env.ts

```diff
@@ -53,6 +53,13 @@ const env = envalid.cleanEnv(process.env, {
     default: '',
     devDefault: 'http://www.example.com/authz',
   }),
+  IDP_EXTERNAL_CREDENTIAL_SECRETS: envalid.str({
+    devDefault: 'alice:secret,bob:secret,charlie:secret',
+  }),
+  IDP_OWNERS: envalid.str({
+    devDefault:
+      '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY,5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty,5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y',
+  }),
```

Comment:
I think we should run passed @dblane-digicatapult how best to do this. My temptation, as these are potentially unbounded lists, is to move them into a separate structured `json` file that we can then reference by environment variable. We should also structure them to guarantee they are passed as triples. 

If we do the `json` file thing we'll need to modify the helm chart to template the values into a secret and then mount the secret as a file

Reply:
We could go down the JSON file route or do something like this in values.yaml:

```
idpCredentials:
  - username: alice
    secret: secret123
    owner: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
  - username: bob
    secret: secret456
    owner: 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
  - username: charlie
    secret: secret789
    owner: 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y
 ```
 
And then range over the list or something?

```
{{- range $index, $item := .Values.idpCredentials }}
  {{- if $index }},{{ end }}
  {
    "username": "{{ $item.username }}",
    "secret": "{{ $item.secret }}",
    "owner": "{{ $item.owner }}"
  }
{{- end }}
```

Then its just a case of mounting the file as a volume.

Reply:
yep just do it as a secret instead of a configmap. You can still mount secrets